src/modules/song/types.ts

```diff
@@ -185,6 +195,9 @@ export interface GetCollaborationsRequest {
   readonly statuses?: CollaborationStatus[];
 }
 
+/**
+ * Collaboration as stored in the NEWM API.
```

Comment:
I like this idea, to add comments to the interfaces that belong to an endpoint, it would make it easier to update in the future 🤷🏽‍♂️